src/components/LayOut/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { NavBar } from "../NavBar/index.jsx";
+import { Footer } from "../Footer/index.jsx";
+import styles from "./layOut.module.css";
+
+export function LayOut({ children }) {
+  return (
+    <div className="container">
```

Comment:
Please use css modules with encapsulation
```suggestion
    <div className={styles.container}>
```

Reply:
done